packages/main/src/Button.hbs

```diff
@@ -27,6 +27,7 @@
 			class="ui5-button-icon"
 			name="{{icon}}"
 			part="icon"
+			aria-hidden="true"
```

Comment:
Not a good practice to add aria attributes on custom element level.

Reply:
I think that the issue is resolved without that

Reply:
@elenastoyanovaa take a look at #4509

Reply:
The axe tool reports "interactive controls must not be nested" without that attribute.
Have a look at the following PR: https://github.com/SAP/ui5-webcomponents/pull/3511
where this solution was applied.

Reply:
@unazko 

This part is not needed. Removing the tabindex=-1 was enough. Adding aria-hidden directly on the ui5-icon was a workaround for the same issue, which was fixed with #4509 . If we now remove this aria-hidden in the checkbox, the issue will not be present as well.

Please test it our yourself:
- Open https://sap.github.io/ui5-webcomponents/playground/main/pages/Button/ . Run the axe tool. The issue reported will be present 
- Now open https://sap.github.io/ui5-webcomponents/master/playground/main/pages/Button/ (this is the version with the merged fix by Pirin) Run the axe tool. The issue will not be there. There is no need of this PR.

Reply:
You are right @elenastoyanovaa. The issue is now fixed as a result of the following PRs:
#3511 - providing default value <code>undefined</code> for the <code>effectiveAccessibleName</code> property, #4509 - removing <code>tabindex=-1</code> from the ui5-icon svg tag. Therefore I'm closing the current PR.